src/components/Input/Input.tsx

```diff
@@ -13,49 +13,18 @@ interface InputProps extends Omit<React.InputHTMLAttributes<HTMLInputElement>, '
     iconRight?: React.ReactNode;
 }
 
-const StyledInputContainer = styled.div`
+const StyledInputContainer = styled.div<Pick<InputProps, 'size' | 'view' | 'brick'> & { focused?: boolean }>`
     position: relative;
     display: flex;
     align-items: center;
-`;
-
-const StyledIconContainer = styled.div<{ size: InputProps['size']; position: 'left' | 'right' }>`
-    position: absolute;
-
-    ${({ size }) =>
-        size === 'm' &&
-        `
-            top: 5px;
-        `}
-
-    ${({ size, position }) =>
-        size === 'm' &&
-        position === 'left' &&
-        `
-            left: 9px; // 8 + 1
-        `}
-
-    ${({ size, position }) =>
-        size === 'm' &&
-        position === 'right' &&
-        `
-            right: 9px; // 8 + 1
-        `}
-`;
-
-// eslint-disable-next-line @typescript-eslint/no-unused-vars
-const StyledInput = styled(({ forwardRef, size, view, brick, iconLeft, iconRight, ...props }: InputProps) => (
-    <input ref={forwardRef} {...props} />
-))`
     box-sizing: border-box;
-    width: 100%;
 
-    font-weight: 500;
+    gap: 9px;
```

Comment:
Hmm... what the number?)

Reply:
Same question about this prop: `padding-left: 30px; // 8 + 8 + 15 - 1` 😄 

But based on this calculation a gap's value must be `8px`